stubs/oauthlib/oauthlib/oauth2/rfc6749/tokens.pyi

```diff
@@ -0,0 +1,59 @@
+from typing import Any
+
+class OAuth2Token(dict[Any, Any]):
```

Comment:
mypy is ok with `dict` typing here, but pyright wants type arguments. mypy won't accept `dict[..., ...]`. I noticed the maintainer's doc mentions favouring `dict` ('use builtin generics'). What's the best thing to do here - `Dict`?

Reply:
If using `dict[Any, Any]` doesn't work, use `Dict[Any, Any]`. (There are some fixes for built-in generics in mypy HEAD that haven't been released yet.)

Reply:
Alright, thanks!